add copyright headers

robjloranger opened the pull request:
adds basic copyright headers to source files

closes #1

Commit:
add copyright headers

ghost commented:
@thebaer what do you think of this format? should we add something like: ` ... and contributors`? Sort of like go has `the go authors`.

Commit:
adds support for filename parsing in zips

this adds a helper to extract collection, slug and id from filenames in
writefreely export zips

Commit:
Revert "adds support for filename parsing in zips"

This reverts commit 826b142327e62d563a357874e66a5f554cde664a.

ghost commented:
whoops, wrong branch

thebaer commented:
@robjloranger yeah let's add that clause. Also did any of this code come from stuff we wrote in 2018? If not, we can remove that year from the notices.

ghost commented:
Ok I'll do that soon, and this was all new code so we can change the first year to 2019.

Commit:
update year and add contributors mention

thebaer commented:
Looks great, thanks!
